pkg/uwb-core/include/dpl/queue.h

```diff
@@ -0,0 +1,525 @@
+/*
```

Comment:
Where does this file come from ? Would it possible to not add it to the code base ?

Reply:
It comes from the package, I can't add it isolated without creating an include nightmare because of where it is located in that package.

Reply:
see 2ce77cd185816fcc8e5529992b22d87df08cc38d